### PR TITLE
Add withdraw NFT

### PR DESCRIPTION
**Pull request by diegoalzate**
closes https://github.com/hoprnet/hopr-admin/issues/389

### ui in safe
CleanShot 2023-09-12 at 16 35 21

### ui in withdraw
CleanShot 2023-09-12 at 16 35 44

### CODE REVIEW

**Review by mjadach-iv — CHANGES_REQUESTED** [review]

**Comment by diegoalzate** [comment]
@mjadach-iv found a bug in current implementation. We have to be able to know when we wan to force and when we basically can not (Authentication error).

I was thinking about splitting these types of errors to `FETCH_ERROR` (<-- i think we can force node admin when fetch fails), `API_ERROR` (this is an error sdk has caught it includes authentication errors) and `NOT_ELIGIBLE_ERROR` ( this is one of those errors we create and we can force login)

Basically, force login when we know the user can authenticate and not checking if they are elegible.
